docs/advanced/verifying-binaries.md

```diff
@@ -197,9 +200,12 @@ WFOp0iCvCqsdMXqjaZUrv2HYCpmpuLkhII9z8y8=
 
 </details>
 
-Simply copy that entire text, save it as key.txt, and then:
-```console
-$ cd \path\to\your\text\key.txt
+Simply copy that entire text, save it as key.txt, and open a terminal.
+
+(Note that Windows uses `\` for directory paths, but Linux/macOS use `/`).
+
+```
+$ cd /path/to/your/text/key.txt
```

Comment:
this instruction will cause confusion written this way. you can't cd to a regular file, only a directory.

Reply:
Good catch. Fixed.